solver/jobs.go

```diff
@@ -143,6 +201,16 @@ func (j *job) loadInternal(def *pb.Definition, resolveOp ResolveOpFunc) (*Input,
 			if err != nil {
 				return nil, err
 			}
+			if opMetadata.IgnoreCache {
+				for i := range pbOp.Inputs {
```

Comment:
Why does it invalidate the inputs for the vertex instead of the vertex itself. Should it read the metadata for the input in here? Or do we need a way to count the number or outputs vertex has(there used to be something like this).